lib/src/component/abstract_transition.dart

```diff
@@ -187,16 +190,19 @@ abstract class AbstractTransitionComponent<T extends AbstractTransitionProps, S
       skipCount = 0;
     }
 
```

Comment:
@greglittlefield-wf Do you think the extraneous warnings still appearing could be resolved by placing `_cancelTransitionEndTimer()` at the top of the `handlePreShowing` method?

 I tried it and it seems to work, but I'm not sure if doing this defeats the purpose of the timer in the first place.